### ISSUE TITLE
db.query does not execute for remote database but does for local database

### ISSUE THREAD

**Issue by davidpharrington** [hgourvest/node-firebird#225]
I can connect to the remote database using ISQL and RazorSQL.
db.query works (prints the output of the select to my console) when I connect to local machine's database, but prints nothing when I query the remote database; it hangs.

var Firebird = require('node-firebird')

//local firebird database
var options = {
    host: '209.xxx.xxx.xxx', 
    port: '3050',
    database: 'E:\\folder\\folder\\abc.fdb',
    user: 'SYSDBA',
    password: 'masterkey',
    lowercase_keys: false,
    pageSize: 4096
};

Firebird.attach(options, function (err, db) {
    if (err) {
        console.log(err)
        throw err;
    }
    db.query('SELECT * from Customer', function (err, result) {
	console.log("debug this")
        console.log(result)
        console.log(err)
        db.detach();
    });

});

---------------------------------------------------------------
>node firebirdconnect.js
//no output appears; it just waits.
